src/Components/ComponentRebuildRelationRequestModel.cs

```diff
@@ -11,6 +11,7 @@ namespace Vasont.Inspire.Models.Components
     /// <summary>
     /// This class contains properties and information to request the system to start a rebuild relation process.
     /// </summary>
+    [Obsolete("Use local DTOs", true)]
```

Comment:
Should we be putting the name of the DTO that's replacing this model in the Obsolete message?